src/styles/index.scss

```diff
@@ -1,3 +1,131 @@
+html {
+  font-family: Roboto, Arial, Helvetica, sans-serif;
+
+  --border-radius: 5px;
+  --review-numb: #060b35;
+  --buy-button: #00acdc;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  display: flex;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: var(--border-radius);
+  flex-direction: column;
+  width: 200px;
+}
+
+.image {
+  display: flex;
+  width: 160px;
+  height: 134px;
+  margin: 32px auto 0;
+}
+
+.product-name {
+  display: flex;
+  color: var(--review-numb);
+  padding-inline: 16px;
+  margin-top: 40px;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+}
+
+.product-code {
+  display: flex;
+  color: #616070;
+  line-height: 14px;
+  margin-top: 4px;
+  font-size: 10px;
+  padding-left: 16px;
+  font-weight: 400;
+}
+
+.review {
+  display: flex;
+  padding-inline: 16px;
+  margin-top: 16px;
+  align-items: flex-end;
+  justify-content: space-between;
+}
+
+.review__stars {
+  display: flex;
+  align-items: center;
+}
+
+.review__stars-star {
+  display: flex;
+  background-color: url(/src/images/star.svg);
```

Comment:
The `background-color` property should be `background-image` to correctly apply the image as a background. Also, verify that the path `/src/images/star.svg` is correct and that the image exists at this location.